docs/source/documentation/diagrams/architecture-diagrams.html.md.erb

```diff
@@ -0,0 +1,33 @@
+# Data Platform - Architectural Diagrams
+
+## C4 Model Diagrams
+
+In the [C4 model](https://c4model.com/), the four C's represent [context](https://c4model.com/#SystemContextDiagram), [containers](https://c4model.com/#ContainerDiagram), [components](https://c4model.com/#ComponentDiagram), and [code](https://c4model.com/#CodeDiagram). These different levels help you to tell the right story to the right audience.
+
+### Level 1 - Context Diagram
+This is the highest level of abstraction and provides an overview of the entire system. It shows the system under consideration and its external dependencies, such as users, external systems, and APIs. Frequency in change will be minimal compared to other levels.
```

Comment:
Full-blown pedantry - just eradicating passive voice:

'Frequency in change is likely minimal compared to other levels.'